test/gtest/uct/test_event.cc

```diff
@@ -109,6 +110,9 @@ void test_uct_event_fd::test_recv_am(bool signaled)
     recv_buffer = (recv_desc_t *) malloc(sizeof(*recv_buffer) + sizeof(send_data));
     recv_buffer->length = 0; /* Initialize length to 0 */
 
+    /* give a chance to finish connection for some transports (ib/ud, tcp) */
+    short_progress_loop(1000);
```

Comment:
why is it needed if you use UCT_TEST_CALL_AND_TRY_AGAIN for sending?

Reply:
the code below
```
wakeup_fd.events = POLLIN;
EXPECT_EQ(0, poll(&wakeup_fd, 1, 0));
```
fails for TCP with nob-blocking `connect()`, because `poll` returns events that are ready to be read.

Reply:
need to make some other check then, because this one does not guarantee the connections will be established during the progress (so the test can fail under high load). Maybe do flush or short send and make sure it is completed.

Reply:
yes, we need something deterministic here. your suggestion to do flush/short send makes sense to me. thank you!
will fix by adding flush() here and other places as well (e.g. `test_pending.cc`)